### ISSUE TITLE
Use custom marquee

### ISSUE THREAD

**Issue by frm** [subvisual/alchemyconf.com#17]
The `marquee` tag is sluggish on Firefox, especially on large screens.

We should instead add a custom marquee component that does the same effect. Preferably one that has an infinite closed loop, instead of leaving an empty space between elements like the native `marquee` tag allows.

**Comment by mirhamasala** [comment]
@frm The `marquee` tag is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee). Perhaps that's why it doesn't work so well.

**Comment by frm** [comment]
Yeah, they're not maintained, they're just kept for nostalgia. The day `blink` and `marquee` are removed, a little piece of the internet dies 😢 

Either way, we're just suckers for these retro details on websites. Unfortunately, we'll have to add a custom element, the performance on Firefox is abysmal.
